proj4/src/main/scala/geotrellis/proj4/io/wkt/WKTParser.scala

```diff
@@ -0,0 +1,152 @@
+
+  def double: Parser[Double] =("""[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?""".r | """[-+]?[0-9]*\.?[0-9]+""".r) map (_.toDouble)
+
+  def int: Parser[Int] = """[-+]?[0-9]+""".r map (_.toInt)
+
+  def value: Parser[Any] = string | double | int
+
+  def comma: Parser[Any] = ("," ~ " ".?) map {
+    case x ~ _ => None
+  }
+
+  def parameter: Parser[Parameter] = """PARAMETER[""" ~ string ~ comma ~ double ~ """]""" map {
+    case _ ~ name ~ _ ~ value ~ _=> new Parameter(name, value)
+  }
+
+  def authority: Parser[Authority] = """AUTHORITY[""" ~ string ~ comma ~ string ~ """]""" map {
+    case _ ~ x ~ _~ y ~ _ => new Authority(x, y)
+  }
+
+  def spheroid: Parser[Spheroid] = """SPHEROID[""" ~ string ~ comma ~ (double | int) ~ comma ~ double ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ axis ~ _ ~ flattening ~ _ ~ authority ~ _ => new Spheroid(name, axis, flattening, authority)
+  }
+
+  def toWgs: Parser[ToWgs84] = """TOWGS84[""" ~ (double <~ ", ".?).* ~ """]""" map {
+    case _ ~ x ~ _ => new ToWgs84(x)
+  }
+
+  def datum: Parser[Datum] = """DATUM[""" ~ string ~ comma ~ spheroid ~ comma.? ~ toWgs.? ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ spheroid ~_ ~ toWgs ~ _ ~ authority ~ _ => new Datum(name, spheroid, toWgs, authority)
+  }
+
+  def primeM: Parser[PrimeM] = """PRIMEM[""" ~ string ~ comma ~ double ~ comma.? ~ authority.? ~ """]""" map {
+    case _ ~ name ~ _ ~ longitude ~ _ ~ authority ~ _ => new PrimeM(name, longitude, authority)
+  }
+
+  def direction: Parser[String] = "NORTH" | "SOUTH" | "EAST" | "WEST" | "UP" | "DOWN" | "NORTH_EAST" | "GEOCENTRIC_X" | "GEOCENTRIC_Y" | "GEOCENTRIC_Z" | "NORTH_WEST"| "\"North along 30 deg East\"" | "\"North along 120 deg East\"" | "North along 165 deg East\"" | "\"North along 75 deg East\"" | "\"North along 45 deg East\"" | "\"North along 135 deg East\"" | "\"North along 45 deg West\"" | "\"North along 105 deg East\"" | "\"North along 15 deg East\"" |"\"North along 15 deg West\"" | "\"North along 75 deg East\"" | "\"North along 45 deg East\"" | "\"North along 15 deg East\"" | "\"North along 75 deg West\"" | "\"North along 15 deg West\""| "\"North along 160 deg East\""| "\"North along 105 deg West\"" | "\"North along 45 deg West\"" | "\"North along 135 deg West\"" |"\"North along 70 deg East\"" | "\"South along 180 deg\"" | "\"North along 90 deg East\"" | "\"North along 0 deg\"" | "\"North along 90 deg East\"" | "\"North along 0 deg\"" | "\"South along 90 deg West\"" | "\"North along 135 deg West\"" | "\"North along 135 deg East\"" | "\"South along 10 deg West\"" | "\"South along 80 deg East\"" | "\"South along 90 deg East\"" | "\"South along 108 deg East\"" | "\"South along 75 deg West\"" | "\"South along 165 deg West\"" | "\"South along 162 deg West\"" | "\"South along 180 def\"" | "\"South along 60 deg West\"" | "\"South along 30 deg East\"" | "\"South along 57 deg East\"" | "\"South along 147 deg East\"" | "OTHER" ^^ {_.toString}
```

Comment:
Remove unused parser, 'symbol' is used instead because there is too much variance in this field despite the spec.